lib/newrelic/elasticsearch.rb

```diff
@@ -19,22 +19,31 @@
     NewRelic::Agent::MethodTracer.extend(NewRelic::Agent::MethodTracer)
 
     ::Elasticsearch::Transport::Client.class_eval do
-      def perform_request_with_new_relic(method, path, params={}, body=nil)
+      @@old_version = false
```

Comment:
Could this be shortened to `@old_version = method(:perform_request).parameters.length == 4` instead?

Reply:
Done
Thanks